client/layout/style.scss

```diff
@@ -28,6 +28,34 @@
 	box-sizing: border-box;
 	overflow: hidden;
 
+	// Turn Reset 'buttons' into regular text links
```

Comment:
Should these just be part of Calypso's main styles?

https://github.com/Automattic/wp-calypso/blob/2cda95b4524e0f8157f6903776a0cb2783ff2c16/client/assets/stylesheets/_main.scss